client/my-sites/plans/jetpack-plans/product-store/more-info-link/index.tsx

```diff
@@ -8,15 +8,20 @@ import './style.scss';
 export const MoreInfoLink: React.FC< MoreInfoLinkProps > = ( { item, onClick } ) => {
 	const translate = useTranslate();
 
-	const isExternalProduct = EXTERNAL_PRODUCTS_LIST.includes( item.productSlug );
+	const isExternalProductLink =
+		EXTERNAL_PRODUCTS_LIST.includes( item.productSlug ) && item.externalUrl;
 
-	const href = isExternalProduct && item.externalUrl ? item.externalUrl : `#${ item.productSlug }`;
+	const href = isExternalProductLink ? item.externalUrl : `#${ item.productSlug }`;
+
+	const target = isExternalProductLink ? '_blank' : '_self';
```

Comment:
Better to use target only when it's external, otherwise `Button` component adds `rel="noopener noreferrer"` even if `target` is set to `_self`.

```suggestion
	const target = isExternalProductLink ? '_blank' : undefined;
```

Reply:
thanks for pointing this @manzoorwanijk 👍